src/main/java/org/jabref/gui/entryeditor/LatexCitationsTabViewModel.java

```diff
@@ -160,19 +170,41 @@ private List<Path> searchDirectory(Path directory) {
         }
     }
 
+    private void setListener(Path path) {
+        if (!hasListener) {
+            try {
+                fileUpdateMonitor.addListenerForFile(path, this::refreshLatexDirectory);
+                hasListener = true;
+            } catch (IOException e) {
+                LOGGER.error("Could not find file", e);
+            }
+        }
+    }
+
+    private void setListener(Path oldPath, Path newPath) {
```

Comment:
The method should be called `updateListener`, because it also removes the old one.